packages/expo/src/Expo.ts

```diff
@@ -15,6 +15,8 @@ export {
   // Methods
   requireNativeModule,
   requireOptionalNativeModule,
+  requireNativeViewManager,
+  registerWebModule,
```

Comment:
Could you open a separate PR with the changes in `expo` package? I would also suggest to export `requireNativeViewManager` as `requireNativeView`. "view manager" is a term from RN old architecture so it would be nice to get rid of it.